core/src/main/java/org/apache/druid/java/util/common/StringUtils.java

```diff
@@ -360,4 +361,54 @@ public static String encodeBase64String(byte[] input)
   {
     return BASE64_DECODER.decode(input);
   }
+
+  /**
+   * Returns a string whose value is the concatenation of the
+   * string {@code str} repeated {@code count} times.
+   * <p>
+   * If count or length is zero then the empty string is returned.
```

Comment:
nit: If count or the length of the string...

Reply:
@QiuMM Thanks :+1:

Reply:
Fixed.